src/sql/workbench/parts/notebook/browser/notebook.css

```diff
@@ -136,4 +136,76 @@
 .notebookEditor .book-nav .dialog-message-button {
 	min-width: 100px;
 	margin-right: 10px;
-}
\ No newline at end of file
+}
+
+.notebookEditor .hoverButtonsContainer {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	width: 40%;
+	margin: 1px auto;
+}
+
+.notebookEditor .hoverButtonsContainer .containerBackground {
+	position: absolute;
+	width: inherit;
+	height: 2px;
+	border-radius: 3px;
+	opacity: 0.4;
+	z-index: 0;
+}
+
+.notebookEditor .hoverButtonsContainer .hoverButton {
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: center;
+	margin: 3px;
+	padding: 4px 6px;
+	font-size: 10px;
+	box-sizing: border-box;
+	border-color: transparent;
+	border-width: 1px;
+	border-style: solid;
+	border-radius: 3px;
+	z-index: 1;
+	visibility: hidden;
+}
+
+.notebookEditor .hoverButtonsContainer:hover .hoverButton {
+	visibility: visible;
+}
+
+.notebookEditor .hoverButton:active {
+	transform:scale(1.05);
+}
+
+.notebookEditor .hoverButton .addCodeIcon,
+.notebookEditor .hoverButton .addTextIcon {
+	display: inline-block;
+	background-position: center center;
+	background-repeat: no-repeat;
+	width: 10px;
+	height: 10px;
+	background-size: contain;
+	margin-right: 4px;
+}
+
+.notebookEditor .hoverButton .addCodeIcon {
+	background-image: url("./media/light/add_code.svg");
```

Comment:
please check with Anthony on relative links, he's been making a lot of changes around this in his code layering.

Reply:
@anthonydresser Any issues we might need to worry about here?

Reply:
I can't think of any.